pkg/externalcluster/externalcluster.go

```diff
@@ -0,0 +1,51 @@
+package externalcluster
```

Comment:
This is not a large chunk of code and we could just keep it in the controller.go file